test/scale-infra/README.md

```diff
@@ -0,0 +1,134 @@
+
+    We advise to use the Ubuntu Amazon EC2 AMI Locator to acquire the AMI value for a specific region: https://cloud-images.ubuntu.com/locator/ec2/. Select Ubuntu release `22.04` or `24.04`. 
+
+    Also, use the flag `--aws-key-name` to specify the name of the uploaded public ssh key to your AWS account in the target region. 
+
+    Upon completion of the script's execution, an Ansible inventory file containing the IP addresses of the nodes that constitute the Kubernetes cluster will be generated at the current directory at `.data/hosts_core`.
+
+
+2. Create WEC hosting instances:
+    
+    Set the following variables:
+    ```console
+    CLUSTER_NAME="wec"
+    NUM_HOSTING_INSTANCES=1
+    EC2_INSTANCE_TYPE="t2.xlarge"
+    ```
+
+    Then, deploy the WEC infra:
+
+    ```bash
+    ./deploy_wec_infra.sh --region $REGION --vpc-name $VPC --wecs-hosting-instances $NUM_HOSTING_INSTANCES --instances-type $EC2_INSTANCE_TYPE --aws-key-name $EC2_SSH_PUBLIC_KEY --arch $ARCH --ec2-image-id $EC2_AMI
+    ```
+
+    Use the flag `--wecs-hosting-instances` to specify the number of ec2 instances to be created to host the WECs. You must create the WEC hosting instances in the same region as the KS control plane hosting infra created a step 1 - multiple regions deployment is not supported at the moment.  
+
+    Upon completion of the script's execution, an Ansible inventory file containing the IP addresses of the ec2 WEC hosting instances will be generated in the present directory at `.data/hosts_wec`.
+
+3. Create WEC kind clusters:
+
+    a) Update the inventory of the WEC instances to include the IP address of a master node from the K8s cluster created in step 1 above:
+
+    First, find the required master's IP address info in the following file: `.data/hosts_core`.
+
+    Next, use the following command to see the content of the WEC ansible inventory:
+    ```bash
+    cat .data/hosts_wec
+    ```
+
+    Sample output: 
+    ```console
+    [masters]
+    <add master node info here!>
+
+    [add_workers]
+    worker1 ansible_host=192.168.56.1
+    ```
+
+    Lastly, edit the contents of the file `.data/hosts_wec` to include the IP address and name of a master node in the [masters] section, for example:
+    ```console
+    [masters]
+    master1 ansible_host=192.168.56.20
+
+    [add_workers]
+    worker1 ansible_host=192.168.56.1
+    ```
+
+    b) Create WEC kind clusters and connect to KS Core cluster
+
+    ```bash
+    ansible-playbook -i .data/hosts_wec deploy_ks_wec.yaml --ssh-common-args="-o StrictHostKeyChecking=no" -e "region=$REGION vpc_name=$VPC num_wecs=1"
+    ```
+
+    Use the input paramater `num_wecs` to specify the number of kind clusters to be created for each WEC Hosting Instances. The above command creates kind WEC clusters and connects them to the KubeStellar core cluster created in step 1. Furthermore, it attaches a [KWOK](https://github.com/kubernetes-sigs/kwok) fake node to each kind cluster. 
+
+
```

Comment:
The setup instructions should include telling the reader what kubeconfig contexts have been created and how to added the missing relevant ones (if any). Same for INSTRUCTIONS.md.

Reply:
The setup instructions and (ideally) the picture should say something about ITS and WDS.

Reply:
A little more broadly, the reader should be told how to manipulate the core and WECs.
They are in a VPC and serve TLS certificates that assert only private and inside-VPC IP addresses.

Reply:
The user does not need to add any kubeconfig context  - all required are included by default. Additionally, we don't want to over-explaining and make the instructions and picture hard to read. The expectation is that a user following these instructions has already some advance knowledge of KubeStellar and Kubernetes (not a novice user).

Reply:
(A) After my last, and successful, test I did a `kubectl config get-contexts` and did not find _any_ related to this PR. That's because I was using my regular kubeconfig file, not `.data/admin.conf`. Without being told about it, a reader will likely be mystified.

(B) After I discovered that I should be using `.data/admin.conf`, I tried and failed. The apiserver IP addresses in there only work rom within the VPC. Are you assuming some VPN connection?

Reply:
I just updated the instructions and pushed a new commit addressing all your comments above (A), (B) & (c)